### PR TITLE
[Merged by Bors] - Fix typos in docs

### PR DESCRIPTION
**Pull request by sbernauer**
## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/demos/trino-taxi-data.adoc [hunk 1]
```diff
@@ -241,7 +241,7 @@ returns
         2 | 2021-08-03 11:51:58.000 | 2021-08-03 12:09:29.000 |           17 |            96.0 |          1.56 | Credit card  |        11.5 |       2.22 |        17.02 
 ----
 Pretty cheap for that amount of persons!
```
Comment (maltesander) [inline docs/modules/ROOT/pages/demos/trino-taxi-data.adoc:243]:
```suggestion
Pretty cheap for that amount of people!
```

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by sbernauer** [comment]
Thanks for the fast review!

**Pull request closed without merging**